

\documentclass{article}
\usepackage{graphicx}
\bibliographystyle{plos2009}

\usepackage{lineno}
\linenumbers

\begin{document}

\title{Using hidden Markov model emission probabilities as a general feature for the taxonomic classification of sequences}


\maketitle


\abstract{Abstract.}

\section{Introduction}

\subsection{Feature extraction and the bias variance tradeoff}
\subsection{The challenge of highly divergent sequences}
\subsection{Homology and compositional based methods}
\subsection{Learning methods}

\section{Methods}


\subsection{Genelearn - modular software}
\subsection{Reftree- a search method for taxonomically structured data}
\subsection{Kmer feature extraction}
\subsection{Emission probability feature extraction}
\subsection{combining homology into composition}
\subsection{Learning algorithms ? Logistic regression/ SVM}
\subsection{GraphLab and scikit learn}
\subsection{Precision recall calculations}

\section{Results}
\subsection{Precision recall of Kmer  vs Genemark vs Genemark + kmer for viruses and multiclass}
\subsection{Test of kmer length}
\subsection{F1 vs length of contig}
\subsection{Taxon level (supplemental data)}
\subsection{Solver comparison (supplemental)}
\subsection{Real metagenomic data: RdRP containing contigs}

\section{Discussion}
\subsection{The importance of feature selection (sparse vs dense, information content multiple sources of information)}
\subsection{Feature selection and signal to noise ratio}
\subsection{Homology free methods for highly divergent samples}



%\bibliography{2016-genelearn}

\end{document}
